R/DataSurvival.R

```diff
@@ -125,7 +125,10 @@ as_stan_list.DataSurvival <- function(object, ...) {
     rownames(design_mat) <- NULL
 
     # Parameters for efficient integration of hazard function -> survival function
-    gh_parameters <- statmod::gauss.quad(n = 15, kind = "legendre")
+    gh_parameters <- statmod::gauss.quad(
+        n = getOption("jmpost.gauss_quad_n"),
+        kind = getOption("jmpost.gauss_quad_kind")
```

Comment:
I don't think we can use something else than `legendre`, because otherwise the integration boundaries change and we would need to adjust computations on the Stan side. Just the `n` option is sufficient